src/main/java/linenux/model/Schedule.java

```diff
@@ -60,13 +82,13 @@ public void clear() {
      * @return List of {@code Task} matching the keywords.
      */
     public ArrayList<Task> search(String[] keywords) {
-        ArrayList<String> keywordsList = new ArrayListUtil.ChainableArrayListUtil<String>(keywords)
+        ArrayList<String> keywordsList = new ArrayListUtil.ChainableArrayListUtil<>(keywords)
                                                           .map(String::toLowerCase)
                                                           .value();
 
-        return new ArrayListUtil.ChainableArrayListUtil<Task>(this.taskList)
+        return new ArrayListUtil.ChainableArrayListUtil<>(this.taskList)
```

Comment:
For this angle bracket, I am not sure removing it will give the same functionality. Might be object class instead. Haave to ask yi hang.